CHANGELOG.rst

```diff
@@ -4,6 +4,24 @@ Kubernetes Collection Release Notes
 
 .. contents:: Topics
 
+v5.0.0
+======
+
+Release Summary
+---------------
+
+This major release drops support for ``ansible-core<2.15``.
+
+Minor Changes
```

Comment:
somehow, #744 is not included in the changelog. Assuming that the reason for that is that PR prepared earlier than #741 merged and this PR wasn't updated